docs/ldml/tr35-info.md

```diff
@@ -895,14 +895,46 @@ The factor and offset can be simple expressions, just like the values in the uni
+
+The allowed attributes are the following:
 
 Attribute Value | Description
 ------------ | -------------
-_si_ | the _International System of Units (SI)_
-_metric_ | a superset of the _si_ units, with some non-SI units accepted for use with the SI or simple multiples of metric units, such as pound-metric (= ½ kilogram)
-_ussystem_ | the inch-pound system as used in the US, also called _US Customary Units_
-_uksystem_ | the inch-pound system as used in the UK, also called _British Imperial Units_, differing mostly in units of volume
+`si` | the _International System of Units (SI)_ See [NIST Guide to the SI, Chapter 4: The Two Classes of SI Units and the SI Prefixes](https://www.nist.gov/pml/special-publication-811/nist-guide-si-chapter-4-two-classes-si-units-and-si-prefixes). Examples: meter, ampere
+`si_acceptable` | units acceptable for use with the SI. See [NIST Guide to the SI, Chapter 5: Units Outside the SI](https://www.nist.gov/pml/special-publication-811/nist-guide-si-chapter-5-units-outside-si). Examples: hour, liter, knot, hectare.
+`metric` | a superset of the _si_ units
+`metric_adjacent` | units commonly accepted in some countries that follow the metric system. Examples: month, arc-second, pound-metric (= ½ kilogram), mile-scandinavian
+`ussystem` | the inch-pound system as used in the US, also called _US Customary Units_
+`uksystem` | the inch-pound system as used in the UK, also called _British Imperial Units_, differing mostly in units of volume
+`jpsystem` | traditional units used in Japan
+`astronomical` | additional units used in astronomy. Examples: parsec, light-year, earth-mass
+`person_age` | special units used for people’s ages in some languages. Except for translation, they have the same system as the associated regular units.
+`currency` | currency units, such as curr-usd = US dollar. These are constructed algorithmically from the Unicode currency identifiers, and do not occur in the child elements of `convertUnits`
+`prefixable` | those units that typically use SI prefixes. This can include measures like `parsec` that are not powers of 10 times a combination of SI base units. It allows implementations to group those units together, and to do sanity checks on the prefix+unit combinations, if they choose. However, they are not required to, especially since there is a significant variance in usage: even the term `megafoot` might be acceptable in some contexts.
+
+Over time, additional systems may be added, and the systems for a particular unit may be refined.
+
+#### Derived Unit System
+
+The systems attributes also apply to compound units, and are computed in the following way.
+
+1. The `prefixable` system is only applicable to base_components, and is thus removed
+2. The `number_prefixes`, `dimensionality_prefix`, `si_prefix`, and `binary_prefix` are ignored
+   * Example: systems(square-kilometer) = systems(meter)
+3. Currency units have the `currency` system
+   * Example: systems(curr-usd) = {currency}
```

Comment:
What is `curr-usd`? Is that newly defined somewhere that I missed?

Reply:
It's been defined for a while. Because it is algorithmic, it doesn't show up in the data files.